src/pages/edit/ActionsBar.js

```diff
@@ -172,29 +172,54 @@ const EditBar = ({ dashboard, ...props }) => {
 
     const renderActionButtons = () => (
         <ButtonStrip>
-            <Button primary onClick={onSave} dataTest="save-dashboard-button">
-                {i18n.t('Save changes')}
-            </Button>
-            <Button onClick={onPrintPreview}>
-                {props.isPrintPreviewView
-                    ? i18n.t('Exit Print preview')
-                    : i18n.t('Print preview')}
-            </Button>
-            <Button onClick={toggleFilterSettingsDialog}>
-                {i18n.t('Filter settings')}
-            </Button>
-            {dashboard.id && (
-                <Button onClick={toggleTranslationDialog}>
-                    {i18n.t('Translate')}
+            <Tooltip
+                content={i18n.t('Cannot save this dashboard while offline')}
+            >
+                <Button
+                    disabled={!online}
+                    primary
+                    onClick={onSave}
+                    dataTest="save-dashboard-button"
+                >
+                    {i18n.t('Save changes')}
                 </Button>
+            </Tooltip>
+            <Tooltip>
```

Comment:
why wrap in a tooltip if the tooltip doesn't have any content? same for the other ones below as well

Reply:
ah, this is a custom component, not the ui tooltip. maybe it can be called OfflineTooltip instead?

Reply:
Yeah I was thinking that OfflineTooltip would be a better name.

Reply:
Done